draft-ietf-rats-architecture.md

```diff
@@ -672,26 +672,60 @@ consumes the corresponding conceptual messages as defined in this document.
 
 # Trust Model {#trustmodel}
 
+## Relying Party
+
 The scope of this document is scenarios for which a Relying Party
 trusts a Verifier that can appraise the trustworthiness of
 information about an Attester.  Such trust might come by the Relying
 Party trusting the Verifier (or its public key) directly, or might
 come by trusting an entity (e.g., a Certificate Authority) that is
-in the Verifier's certificate chain.  The Relying Party
-might implicitly trust a Verifier (such as in the Verifying Relying
-Party combination).  Or, for a stronger level of security, the
-Relying Party might require that the Verifier itself provide
+in the Verifier's certificate chain.
+
+The Relying Party
+might implicitly trust a Verifier, such as in a Verifying Relying
+Party combination where a Verifier and a Relying Party are combined.
```

Comment:
```suggestion
might implicitly trust a Verifier, such as in a Verifying/Relying Party 
combination where a Verifier and a Relying Party are combined.
```

Reply:
(I think that we should have a / or something in this new term)

Reply:
Appraising/Relying Party is incorrect because a Relying Party already appraises Attestation Results.